### ISSUE TITLE
dataset structure

### ISSUE THREAD

**Issue by catherine-qian** [Svito-zar/gesticulator#14]
i'm confused about the dataset structure.
I have downloaded the trinity speech gesture dataset, it includes several folders:

  | AlignmentTimes.csv | 2019-10-01 16:20 | 338 |  
-- | -- | -- | -- | --
  | Audio/ | 2019-10-01 15:34 | - |  
  | BVH_from_clap/ | 2020-08-14 19:31 | - |  
  | FBXs/ | 2019-01-23 17:16 | - |  
  | FBXs_from_clap/ | 2020-08-14 21:34 | - |  
  | GENEA_Challenge_2020_data_release/ | 2020-09-09 12:45 | - |  

1. For this repository, shall we only use the GENEA_Challenge_2020_data_release and ignore the others?
2. For GENEA_Challenge_2020_data_realease, we then apply
----------
# encode motion from BVH files into exponensial map representation
python bvh2features.py

# Split the dataset into training and validation
python split_dataset.py

# Encode all the features
python process_dataset.py
---------
to the 'Training_data' and 'Test_data' folders, respectively?
3. I'm confused about the data structure, could the authors shall their data structure?

**Comment by Svito-zar** [comment]
Hi , @catherine-qian .

Yes, you understood it right that you only need the `GENEA_Challenge_2020_data_release` folder and can ignore the others.
I would suggest you use only the `Training_data` folder. The `test_data` folder is supposed to be used neither for training nor for validation during hyper-parameters tuning, but only to test the final model.

Did I answer your question?

**Comment by catherine-qian** [comment]
@Svito-zar thanks svito, that is clear now!

**Comment by catherine-qian** [comment]
Dear Svito, for 'split_data' file
It split the Traning_data into dev(1,2), test (3) and train (4-23) subset, see below:
![image](https://user-images.githubusercontent.com/8257740/102329418-56a8ae80-3fc3-11eb-8054-b60ee5bff6cf.png)


Does the method report in the paper train on the 4-23 (totally 19) sequences?
And test on the single sequence (3)?

**Comment by ghenter** [comment]
Just as a short note, @Svito-zar is on vacation this week and may not be able to respond to this as quickly as usual. Unfortunately, I do not know the specifics of the data split for the paper, so I cannot answer this question myself.

**Comment by Svito-zar** [comment]
Hi, @catherine-qian , that's a good question. Sorry for the late reply.

No, actually, we used a different split for the experiments in our paper. The split provided in this code was following the [GENEA Challenge](https://genea-workshop.github.io/2020/#gesture-generation-challenge) data split, since the data was release specifically for the GENEA Challenge.

I did not expect the results to be significantly different based on the split and hence did not provide the instruction on how to obtain exactly the same split as we have used for our experiments. But I can provided it for you if you find it nessesary.

Best regards, @Svito-zar .
